src/script/interpreter.h

```diff
@@ -101,6 +101,7 @@ enum SigVersion
 {
     SIGVERSION_SPROUT = 0,
     SIGVERSION_OVERWINTER = 1,
+    SIGVERSION_SAPLING = 2,
```

Comment:
This is not mentioned anywhere in ZIP-243.

Reply:
It's mostly an implementation detail, instead of having three separate functions. But the fact that we are switching on the version group ID might be worth mentioning (though that itself is also just an implementation detail of how we are targeting its use on v4 transactions).